components/datatypes/marketing/commerce.schema.json

```diff
@@ -153,6 +153,11 @@
           "title": "Scope",
           "$ref": "https://ns.adobe.com/xdm/datatypes/commercescope",
           "description": "Commerce scope identifiers of where an event occurred (store view, store, website etc.)."
+        },
+        "xdm:commerceChannel": {
+          "title": "Commerce Channel",
+          "type": "string",
```

Comment:
@mjasrotia Here's an example of what I was thinking of, from the Slack conversation. This would be similar to the promotionID field near the top of this file - just a string field directly under the commerce node. We might also do a commerceChannel node and put a couple of fields under it, if there are multiple relevant values we anticipate needing, but I wanted to indicate the general idea first. This shouldn't show up under a tenant namespace as far as I understand.

CC @anandphatak

Reply:
@mjasrotia Will you have time to look at this soon? It's now blocking development and release for this value